workflows/raredisease.nf

```diff
@@ -1,141 +1,627 @@
 
 //
-// MODULE: Installed directly from nf-core/modules
+// SUBWORKFLOWS
 //
-include { FASTQC  } from '../modules/nf-core/software/fastqc/main'  addParams( options: modules['fastqc'] )
-include { MULTIQC } from '../modules/nf-core/software/multiqc/main' addParams( options: multiqc_options   )
+
+include { ALIGN                                 } from '../subworkflows/local/align'
+include { ANALYSE_MT                            } from '../subworkflows/local/analyse_MT'
+include { ANNOTATE_CSQ_PLI as ANN_CSQ_PLI_SNV   } from '../subworkflows/local/annotate_consequence_pli'
+include { ANNOTATE_CSQ_PLI as ANN_CSQ_PLI_SV    } from '../subworkflows/local/annotate_consequence_pli'
+include { ANNOTATE_SNVS                         } from '../subworkflows/local/annotate_snvs'
+include { ANNOTATE_STRUCTURAL_VARIANTS          } from '../subworkflows/local/annotate_structural_variants'
+include { CALL_REPEAT_EXPANSIONS                } from '../subworkflows/local/call_repeat_expansions'
+include { CALL_SNV                              } from '../subworkflows/local/call_snv'
+include { CALL_STRUCTURAL_VARIANTS              } from '../subworkflows/local/call_structural_variants'
+include { CHECK_INPUT                           } from '../subworkflows/local/check_input'
+include { GENS                                  } from '../subworkflows/local/gens'
+include { PREPARE_REFERENCES                    } from '../subworkflows/local/prepare_references'
+include { QC_BAM                                } from '../subworkflows/local/qc_bam'
+include { RANK_VARIANTS as RANK_VARIANTS_SNV    } from '../subworkflows/local/rank_variants'
+include { RANK_VARIANTS as RANK_VARIANTS_SV     } from '../subworkflows/local/rank_variants'
+include { SCATTER_GENOME                        } from '../subworkflows/local/scatter_genome'
+include { PEDDY_CHECK                           } from '../subworkflows/local/peddy_check'
+
 
 /*
-========================================================================================
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     RUN MAIN WORKFLOW
-========================================================================================
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
 // Info required for completion email and summary
 def multiqc_report = []
 
 workflow RAREDISEASE {
 
-    ch_software_versions = Channel.empty()
+    ch_versions = Channel.empty()
 
-    //
-    // SUBWORKFLOW: Read in samplesheet, validate and stage input files
-    //
-    INPUT_CHECK (
-        ch_input
+    // Initialize input channels
+    if (params.input) {
+        ch_input = Channel.fromPath(params.input)
+        CHECK_INPUT (ch_input)
+        ch_versions = ch_versions.mix(CHECK_INPUT.out.versions)
+    } else {
+        exit 1, 'Input samplesheet not specified!'
```

Comment:
Don't use exit

Reply:
Also this one

Reply:
This should also be fixed 😅

Reply:
ok this one is definitely still shown for me when I click "View file", I am then on revision 6023bf6. however, if I change then to `dev` it is not shown 🤯

Reply:
well now it is fine. No idea, maybe cache, maybe github had a hiccup